CHANGELOG.adoc

```diff
@@ -663,6 +663,7 @@ When closed, clicking on any of the views' icon will re-open the panel to make t
 - https://github.com/eclipse-sirius/sirius-components/issues/1201[#1201] [charts] Prepare support for charts in Sirius Components
 - https://github.com/eclipse-sirius/sirius-components/issues/1180[#1180] [diagram] Add support for the dynamic computation of the connector tools to control the tools displayed in the contextual menu.
 - https://github.com/eclipse-sirius/sirius-components/issues/1212[#1212] [form] Add support for styling of the View DSL widgets
+- https://github.com/eclipse-sirius/sirius-components/issues/2080[#2080] [tree] Add direct edit support on trees items.
```

Comment:
Direct edit support on tree items (not trees items) has been there for a long time, this is not what this commit brings. Could you please update the title of the commit and the description of the change here to match what your contribution is improving?

Reply:
Don't forget to describe the API breaks that are coming with this contributions in the dedicated section (such as RepresentationMetadata)